### PR TITLE
v0.5.5 License and Readme changes [skip ci]

### PR DESCRIPTION
**Pull request by mdubya66**
New License file and other changes to project readme's

### CODE REVIEW

File: doc/processes/RELEASE-PROCESS.md [hunk 1]
```diff
@@ -1,103 +1,88 @@
...
-See [CONTRIBUTING.md](../../CONTRIBUTING.md#reporting-bugs-and-creating-issues) for details on how issues are prioritized and tracked.
+See [CONTRIBUTING.md](../../CONTRIBUTING.md#reporting-bugs-and-creating-issues)
+for details on how issues are prioritized and tracked.
 
 ## Release Versioning
 
-The VIC project follows Semantic Versioning 2.0.0. This is described in detail at 
-http://semver.org/
+The VIC Engine project follows Semantic Versioning 2.0.0. This is described in
+detail at http://semver.org/
```
Comment (matthewavery) [inline doc/processes/RELEASE-PROCESS.md]: details*

**Comment by matthewavery** [comment]
lgtm

**Comment by dougm** [comment]
lgtm

**Comment by mdubya66** [comment]
blop

**Pull request merged**